Installing-and-Configuring/Building.md

```diff
@@ -2,29 +2,39 @@
 layout: default
 title: Building Heads
 permalink: /Building/
-nav_order: 8
+nav_order: 2
+parent: Installing and configuring
 ---
 
 Building Heads
 ===
-Heads is supposed to be a [reproducible build](https://reproducible-builds.org/) and as of [v0.1.0](https://github.com/osresearch/heads/releases/tag/v0.1.0) it achieved this goal.  The downside is that the initial build can take a very long time as it downloads and builds all of the its dependencies.  One issue right now is that it builds not just one, but *two* cross compilers and as a result takes about 45 minutes.  Luckily subsequent builds only take about 30 seconds to produce a full coreboot and Linux ROM image, but that first ones a doozy...
+
+Heads is supposed to be a [reproducible build](https://reproducible-builds.org/)
+ and as of [v0.1.0](https://github.com/osresearch/heads/releases/tag/v0.1.0) it
+ achieved this goal.  The downside is that the initial build can take a very
+ long time as it downloads and builds all of the its dependencies.  One issue
+ right now is that it builds not just one, but *two* cross compilers and as a
+ result takes about 45 minutes.  Luckily subsequent builds only take about 30
+ seconds to produce a full coreboot and Linux ROM image, but that first ones a
+ doozy...
 
 With a vanilla Debian 9 or Ubuntu 16.04 install, such as a digitalocean
 droplet, you need to first install some support tools. This takes a
-short while, so get a cup of coffee and [install host build requirements packages as specified here](https://github.com/osresearch/heads/blob/master/.circleci/config.yml#L10-L11)
+short while, so get a cup of coffee and
+[install host build requirements packages as specified here](https://github.com/osresearch/heads/blob/master/.circleci/config.yml#L10-L11)
```

Comment:
I remember you said you were against this? Was my shortcut but maybe it needs to be better informing....

Reply:
Not against it per se but felt that is wasn't descriptive enough to cover different building platforms.  Guix package management would solve this package ambiguity.